src/main.ts

```diff
@@ -92,8 +92,7 @@ export default class ObsidianGit extends Plugin {
                 window.setInterval(() => this.statusBar.display(), 1000)
             );
         }
-        this.app.workspace.onLayoutReady(() => this.init());
```

Comment:
I guess you are on an old version of the api. Try running `npm update`. Your fix is actually deprecated.

Reply:
oh, you're right! I've ran `npm install` and assumed that I am up-to-date :facepalm: